specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,783 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RegistryCredentials"
+          },
+          "x-ms-identifiers": [
+            "server"
+          ]
+        },
+        "dapr": {
+          "$ref": "#/definitions/Dapr",
+          "description": "Dapr configuration for the Container App."
+        }
+      }
+    },
+    "ContainerAppsJob": {
+      "type": "object",
+      "description": "Container App Job",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "extendedLocation": {
+          "$ref": "./CommonDefinitions.json#/definitions/ExtendedLocation"
+        },
+        "identity": {
+          "description": "managed identities for the Container App to interact with other Azure services without maintaining any secrets or credentials in code.",
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity"
+        },
+        "properties": {
+          "description": "ContainerAppsJob resource specific properties",
+          "provisioningState": {
+            "description": "Provisioning state of the Container Apps Job.",
+            "enum": [
+              "InProgress",
+              "Succeeded",
+              "Failed",
+              "Canceled",
+              "Deleting"
+            ],
+            "type": "string",
+            "readOnly": true,
+            "x-ms-enum": {
+              "name": "ContainerAppsJobProvisioningState",
+              "modelAsString": true
+            }
+          },
+          "environmentId": {
+            "description": "Resource ID of environment.",
+            "type": "string",
+            "x-ms-mutability": [
+              "create",
+              "read"
+            ]
+          },
+          "workloadProfileType": {
+            "$ref": "./CommonDefinitions.json#/definitions/WorkloadProfileType",
+            "description": "Workload profile type to pin for container apps job execution."
+          },
+          "configuration": {
+            "$ref": "#/definitions/Configuration",
+            "description": "Non versioned Container Apps Job configuration properties."
+          },
+          "template": {
+            "$ref": "./CommonDefinitions.json#/definitions/ContainerAppsJobTemplate",
+            "description": "Container App versioned application definition."
+          },
+          "outboundIpAddresses": {
+            "description": "Outbound IP Addresses for container apps job.",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "readOnly": true
+          },
+          "eventStreamEndpoint": {
+            "description": "The endpoint of the eventstream of the container apps job.",
```

Comment:
Do we need this @vinisoto since we don't allow incoming http traffic to a job?

Reply:
@trajkobal I think we would require this. Keeping this property for now.